src/sign-in-xplat/README.md

```diff
@@ -0,0 +1,113 @@
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- csharp
+page_type: sample
+name: ".NET MAUI Multi-platform App Sign-in user"
+description: "This is a .NET MAUI Multi-platform App Sign-in user that sign-in users. The code in this sample is used by one or more articles on docs.microsoft.com."
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-csharp
+---
+# .NET MAUI Multi-platform App - Sign-in user | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This sample demonstrates a .NET MAUI Multi-platform App that sign-in users by using the `Microsoft.Identity.Client`.
+
+<!-- IMAGE or CONSOLE OUTPUT of running/executed app -->
+
+> :page_with_curl: This sample application backs one or more technical articles on docs.microsoft.com. <!-- TODO: Link to first tutorial in series when published. -->
+
+## Prerequisites
+
+1. An Azure Active Directory (Azure AD) tenant. You can [open an Azure account for free](https://azure.microsoft.com/free) to get an Azure AD instance.
+1. [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/#download-preview)
+1. [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+1. [.NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation)
+
+## Setup
+
+### 1. Register the web API application in your Azure Active Directory
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app) to register the sample app.
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                         | Notes                                                                                                       |
+|:------------------------------:|:------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
```

Comment:
yeah, it looks nicer @mmacy, but please just let me know in case you want to revert.


done | addressed from [fb8c4](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/15/commits/fb8c4357be35f62bc212c782f9b15efdda3a6204)